CONTRIBUTING.md

```diff
@@ -54,6 +54,28 @@ In addition to the style guide above, client-side Angular components should conf
 
 Projects can be manually linted with `npm run lint`. If you are using Atom and have run the [environment provisioning scripts](https://github.com/treehouseaustin/environment-setup) your IDE comes configured with AtomLinter which will use all project-specific rules. If you are using a different IDE you should configure it with appropriate linting and not rely on the CI process to catch code style issues.
 
+#### Sass-lint
+
+Projects with a css style component will additionally adhere to code style enforced by `sass-lint`. See the [.sass-lint.yml file](./.sass-lint.yml) and reference [sass-lint docs/rules](https://github.com/sasstools/sass-lint/tree/develop/docs/rules) for individual rule explanations.
+
+To include, update your `package.json` to include the following structure:
+```
+{
+  "devDependencies": {
+    "@treehouse/guidelines": "*"
+    "sass-lint": "*"
```

Comment:
I agree that this would be optimal, and certainly possible. However, I would want to make the same changes in how we are using eslint.

Although the eslint module is included in dependencies of this project, that is not the copy that we've been using in the platform project. That project invokes `./node_modules/.bin/eslint ./` through in its script. That copy of eslint is only available in this location because eslint is directly declared as a dependency within platform. Dependencies of dependencies are not linked to in `.bin`.

In order to actually use this repo's version we need to invoke `./node_modules/@treehouse/guidelines/node_modules/.bin/eslint ./`. It's not a problem, just a little verbose. You can verify this by removing eslint from dependencies, delete the copy in `./node_modules/`, re-run `npm install` and run `npm run lint`.

If we're going to do it this way, I would want it to be consistent across eslint, sass-lint, and esw. I have no problem including that in this body of work.